metricbeat/docs/developer-guide/create-metricset.asciidoc

```diff
@@ -209,14 +206,14 @@ metricbeat:
       metricsets: ["{metricset1}", "{metricset2}"]
 ----
 
-NOTE: Make sure to run `make collect` every time after you update the config file
-so it is also applied to the global configuration file and the docs.
+NOTE: Make sure that you run `make collect` after updating the config file
+so that your changes are also applied to the global configuration file and the docs.
 
-More details on the configuration file can be found here in the module docs.
+For more details about the configuration file, see <<configuration-metricbeat>>.
 
 
 [float]
 == Next Steps
```

Comment:
Changed this to "What to Do Next". Not sure if it's better, but maybe it translates better to other languages?